tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature

```diff
@@ -127,20 +127,20 @@ Feature: deleting files and folders
     And file "zzzz-must-be-last-file-in-folder.txt" should not be listed on the webUI
     And no message should be displayed on the webUI
 
-  @skip @yetToImplement @public_link_share-feature-required
+  @public_link_share-feature-required
```

Comment:
if you change the test as you have done below, the `@public_link_share-feature-required` is not needed anymore